src/Settings.php

```diff
@@ -489,6 +489,26 @@ public function __construct()
             );
         }
 
+        // This option should be visible only if the GF Hubspot add-on is activated.
+        $is_gf_hubspot_addon = function_exists('is_plugin_active') && is_plugin_active('gravityformshubspot/hubspot.php');
```

Comment:
For phpunit I guess you could also specify root namespace for the function with`\`: `$is_gf_hubspot_addon = \is_plugin_active('gravityformshubspot/hubspot.php');`, but it's also fine like that :)